### PR TITLE
test: disable Chromatic for VerifyCORS

### PR DESCRIPTION
**Pull request by ktmud**
🏠 Internal

Follow up with #861 

Enable stable tests for more stories.

### CODE REVIEW

File: packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-choropleth-map/Stories.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { SuperChart } from '@superset-ui/core';
+import { withKnobs, select } from '@storybook/addon-knobs';
 import {
   maps,
   ChoroplethMapChartPlugin,
 } from '../../../../../../plugins/plugin-chart-choropleth-map/src';
-import { withKnobs, select } from '@storybook/addon-knobs';
```
Comment (villebro) [inline packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-choropleth-map/Stories.tsx:7]: I'd love for that very long relative import above to be absolute.

File: packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/birthNames.json [hunk 2]
```diff
@@ -522,7 +522,7 @@
           "__timestamp": "1971-01-01T00:00:00",
           "state": "other",
           "gender": "boy",
-          "name": "<img title=\"This is an image\" src=\"https://superset.incubator.apache.org/_images/s.png\" width=\"60\" />",
+          "name": "<img title=\"This is an image\" src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z/C/HgAGgwJ/lK3Q6wAAAABJRU5ErkJggg==\" width=\"60\" />",
```
Comment (villebro) [inline packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/birthNames.json:525]: Not my strong suit, but is this change necessary?

**Review by villebro — APPROVED** [review]
A few non-blocking comments.

File: packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/birthNames.json (hunk 2 above)
Comment (ktmud) [reply packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-table/birthNames.json:525]: The screenshots sometimes mismatch when the image fails to download. This eliminates that possibility.
